packages/xrpl/src/models/transactions/MPTokenIssuanceCreate.ts

```diff
@@ -150,12 +152,26 @@ export function validateMPTokenIssuanceCreate(
     }
   }
 
+  // eslint-disable-next-line @typescript-eslint/consistent-type-assertions -- Not necessary
+  const flags = tx.Flags as number | MPTokenIssuanceCreateFlagsInterface
+  const isTfMPTCanTransfer =
+    typeof flags === 'number'
+      ? isFlagEnabled(flags, MPTokenIssuanceCreateFlags.tfMPTCanTransfer)
+      : flags.tfMPTCanTransfer ?? false
+
   const MAX_TRANSFER_FEE = 50000
   if (typeof tx.TransferFee === 'number') {
     if (tx.TransferFee < 0 || tx.TransferFee > MAX_TRANSFER_FEE) {
       throw new ValidationError(
         'MPTokenIssuanceCreate: TransferFee out of range',
```

Comment:
Specify the range in the error message.
```suggestion
        'MPTokenIssuanceCreate: TransferFee must be between 0 and ' + MAX_TRANSFER_FEE,
```